staging/src/k8s.io/apiserver/pkg/reconcilers/peer_endpoint_lease_test.go

```diff
@@ -91,7 +91,7 @@ func TestPeerEndpointLeaseReconciler(t *testing.T) {
 	newFunc := func() runtime.Object { return &corev1.Endpoints{} }
 	sc.Codec = apitesting.TestStorageCodec(codecs, corev1.SchemeGroupVersion)
 
-	s, dFunc, err := factory.Create(*sc.ForResource(schema.GroupResource{Resource: "endpoints"}), newFunc)
+	s, dFunc, err := factory.Create(*sc.ForResource(schema.GroupResource{Resource: "endpoints"}), newFunc, nil, "")
```

Comment:
Can we set newList at least in places where we set non-nil newFunc?

Reply:
sure, np.